src/main/java/picard/util/LiftoverUtils.java

```diff
@@ -390,20 +390,29 @@ protected static void leftAlignVariant(final VariantContextBuilder builder, fina
                     .map(a -> a.length)
                     .anyMatch(l -> l == 0)) {
                 // 6. extend alleles 1 nucleotide to the left
```

Comment:
This comment should also merge with 5. I'm also not sure what it means to have an "empty allele." Can this have an example in comment?